ha/core/cluster/cluster_manager.py

```diff
@@ -296,3 +304,152 @@ def controller_list(self) -> list:
             [list]: list of controllers.
         """
         return list(self._controllers.keys())
+
+    def get_system_health(self, element: CLUSTER_ELEMENTS = CLUSTER_ELEMENTS.CLUSTER.value, depth: int = 1, **kwargs) -> dict:
+        """
+        Return health status for the requested elements.
+
+        Args:
+            element ([CLUSTER_ELEMENTS]): The element whose health status is to be returned.
+            depth ([int]): A depth of elements starting from the input "element" that the health status
+                is to be returned.
+            **kwargs([dict]): Variable number of arguments that are used as filters,
+                e.g. "id" of the input "element".
+
+        Returns:
+            ([dict]): Returns dictionary. {"status": "Succeeded"/"Failed"/"Partial", "output": "", "error": ""}
+                status: Succeeded, Failed, Partial
+                output: Dictionary with element health status
+                error: Error information if the request "Failed"
+        """
+
+        try:
+            # Currently, only "id" supported in the variable arguments.
+            unsupported_element = True
+            for supported_element in CLUSTER_ELEMENTS:
+                if element == supported_element.value:
+                    unsupported_element = False
+                    break
+
+            # TODO: Fix below temporary code
```

Comment:
Please resolve the TODO  ? or if only node is supported at the moment, modify the TODO accordingly

Reply:
Yes, I will fix this in next PR for the same task. The task will not be closed till then.

Reply:
okay